### ISSUE TITLE
Running 'hence machine connect' is producing a bash error

### ISSUE THREAD

**Issue by homertherefore** [hence-io/cli#4]
Attempting to run the machine connect set up with the export eval, bash is encountering an issue processing the variables.

``` bash
Sean@iMac ~/sites/hence $ eval $(hence machine connect --export)
bash: export: `': not a valid identifier
```

**Comment by ghost** [comment]
Should be fixed in v0.0.12.  Please try again and close on confirmation.

**Comment by homertherefore** [comment]
All clear!
